### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  java.lang.reflect.InvocationTargetException

### ISSUE THREAD

**Issue by NikolayVaklinov10** [microsoft/azure-tools-for-java#3956]
IntelliJ build version: 2019.3 IU-193.5233.102
OS: Mac OS X
JDK: JetBrains s.r.o 11.0.4
Plugin version: 3.30.0-2019.3
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,22,687x789,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at com.microsoft.azure.hdinsight.spark.console.SparkActionPromoterScalaDelegate.promote(SparkActionPromoterDelegate.kt:39)
Caused by: java.lang.AbstractMethodError: Missing implementation of resolved method 'abstract boolean shouldPromote(com.intellij.openapi.actionSystem.AnAction, com.intellij.openapi.actionSystem.DataContext)' of abstract class org.jetbrains.plugins.scala.actions.SingleActionPromoterBase.
```

**Comment by NikolayVaklinov10** [comment]
I am just trying to make a file where to write a code to print Hello World in Scala when I try to save the file the error from above appears

**Comment by wezhang** [comment]
Thanks @NikolayVaklinov10 !

**Comment by wezhang** [comment]
Dup #3864, @NikolayVaklinov10 , the issus has been fixed on v3.31 and above. Could you upgrade to the most new version? Thanks! Feel free to reopen it when you hit it at new version plugin.
